libraries/AC_PID/AC_PID.cpp

```diff
@@ -253,45 +336,40 @@ float AC_PID::get_d() const
 
 float AC_PID::get_ff()
 {
-    _pid_info.FF = _target * _kff;
-    return _target * _kff;
+    return  _pid_info.FF + _pid_info.DFF;
 }
 
 void AC_PID::reset_I()
 {
     _integrator = 0.0;
 }
 
+// load original gains from eeprom, used by autotune to restore gains after tuning
 void AC_PID::load_gains()
 {
     _kp.load();
     _ki.load();
     _kd.load();
     _kff.load();
-    _kimax.load();
-    _kimax.set(fabsf(_kimax));
-    _kpdmax.load();
-    _kpdmax.set(fabsf(_kpdmax));
```

Comment:
Presumable this is because autotune does not change I max and PD max?

Reply:
Yes